src/App.js

```diff
@@ -1,5 +1,5 @@
-import React, { Component, useEffect, useState } from "react";
-import { BrowserRouter, Route, Switch, useHistory } from "react-router-dom";
+import React, { useEffect, useState } from "react";
+import { /* BrowserRouter, */ Route, Switch, useHistory } from "react-router-dom";
```

Comment:
If this is commented we can remove this BrowserRouter instead of commenting it

Reply:
We could, but I was leaving it in there for history and context since we haven't gotten this app working right yet.